Makefile

```diff
@@ -6,7 +6,9 @@
 # K8S_WEBROOT=~/src/github.com/kubernetes/website
 # K8S_ROOT=~/k8s/src/k8s.io/kubernetes
 # K8S_RELEASE=1.17
+# RC_NUM=-rc.3
 
+RCNUM=${RC_NUM}
```

Comment:
instead of introducing additional vars, how about we relax the release number string to be '1.18.0', '1.18.1' etc?

Reply:
Yes, thinking on it.

Reply:
I'd like the option to use the release candidate tag before the release, otherwise
the root string is okay,

Reply:
OK